Mage/src/main/java/mage/abilities/TriggeredAbilityImpl.java

```diff
@@ -120,9 +120,9 @@ public boolean checkUsedAlready(Game game) {
         return lastTurnUsed != null && lastTurnUsed == game.getTurnNum();
     }
 
-    public TriggeredAbility setDoOnlyOnce(boolean doOnlyOnce) {
+    public TriggeredAbility setDoOnlyOnceEachTurn(boolean doOnlyOnce) {
```

Comment:
Interesting that this one is defined only here in the Impl and the other is defined in the Interface